static/components/TaskSummary.jsx

```diff
@@ -77,11 +77,11 @@ var TaskSummary = React.createClass({
     if (task.status === 'failed') {
       className += ' failed';
     } else if (task.status === 'cancelled') {
-      className += ' cancelled';
+      className += 'cancelled';
```

Comment:
if you're pulling the leading space out of here should you also take it out of ` failed`?  or does leaving it on `failed` mean this one shouldn't be removed?